specs/ScreenCaptureRequested.md

```diff
@@ -0,0 +1,348 @@
+} 
+/// This is an extension of the ICoreWebView2Frame interface that supports ScreenCaptureRequested 
+// MSOWNERS: stevenwei@microsoft.com 
+[uuid(12885cda-9caa-4793-9c38-f15827dbab1f), object, pointer_default(unique)] 
+interface ICoreWebView2Frame3 : IUnknown { 
+  /// Add an event handler for the `ScreenCaptureRequested` event. 
+  /// `ScreenCaptureRequested is raised when content in an iframe or any of its 
+  /// descendant iframes requests permission to use the Screen Capture  
+  /// API from getDisplayMedia() 
+  /// 
+  /// This relates to the `ScreenCaptureRequested` event on the  
+  /// CoreWebView2`. 
+  /// Both these events will be raised in the case of an iframe requesting 
+  /// permission. The `CoreWebView2Frame`'s event handlers will be invoked 
+  /// before the event handlers on the `CoreWebView2`. If the `Handled`  
+  /// property of the `ScreenCaptureRequestedEventArgs` is set to TRUE  
+  /// within the`CoreWebView2Frame` event handler, then the event will not  
+  /// be raised on the `CoreWebView2`, and its event handlers will not be   
+  /// invoked. 
+  /// 
+  HRESULT add_ScreenCaptureRequested( 
+      [in] ICoreWebView2FrameScreenCaptureRequestedEventHandler* handler, 
+      [out] EventRegistrationToken* token); 
+  
+  /// Remove an event handler previously added with  
+  /// `add_ScreenCaptureRequested` 
+  HRESULT remove_ScreenCaptureRequested( 
+      [in] EventRegistrationToken token); 
+} 
+ 
+/// Receives `ScreenCaptureRequested` events for iframes. 
+// MSOWNERS: stevenwei@microsoft.com 
+[uuid(c07ac75c-2105-4bb8-9c57-21b6ed8fb381), object, pointer_default(unique)] 
+interface ICoreWebView2FrameScreenCaptureRequestedEventHandler : IUnknown { 
+  /// Provides the event args for the corresponding event. 
+  HRESULT Invoke( 
+      [in] ICoreWebView2Frame* sender, 
+      [in] ICoreWebView2ScreenCaptureRequestedEventArgs * args); 
+} 
+
+```
+
+## C#
+```c#
+namespace Microsoft.Web.WebView2.Core
+{
+    runtimeclass CoreWebView2ScreenCaptureRequestedEventArgs
+    {
+
+        [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2ScreenCaptureRequestedEventArgs")]
+        {
+            CoreWebView2FrameInfo FrameInfo { get; };
+            Boolean Cancel { get; set; };
+            Boolean Handled { get; set; };
+        }
+
+    }
+
+    runtimeclass CoreWebView2
+    {
+        // ...
+        event Windows.Foundation.TypedEventHandler<CoreWebView2, CoreWebView2ScreenCaptureRequestedEventArgs> ScreenCaptureRequested;
+    };
+
+    runtimeclass CoreWebView2Frame
+    {
+        // ...
+        // ICoreWebView2Frame3 members
+        [doc_string("ScreenCaptureRequested is raised when content in an iframe or any of its descendant iframes requests permission to use screen capture.\nIf a deferral is not taken on the event args, the subsequent scripts are blocked until the event handler returns. If a deferral is taken, the scripts are blocked until the deferral is completed.")]

```

Comment:
This line is longer than 80/100 chars, but also the only instance of documentation in the MIDL3. I'd recommend removing this and only having the docs in the COM IDL above to avoid differences between the two and so on.